src/core/server/services/users/auth/invite.ts

```diff
@@ -196,8 +197,7 @@ export async function invite(
     // Check to see if the user with the specified email already has an account.
     const userAlready = await retrieveUserWithEmail(mongo, tenant.id, email);
     if (userAlready) {
-      payloads.push({ email });
-      continue;
+      throw new InviteIncludesExistingUser(email);
```

Comment:
This could possibly create a few invites before failing. Perhaps doing a loop before hand to test each if a user already exists before actually creating the invites?